backend/src/main/java/org/sefglobal/core/model/Certificate.java

```diff
@@ -0,0 +1,52 @@
+package org.sefglobal.core.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Entity
+@Table(name = "certificate")
+public class Certificate {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    @NotNull
+    @Column(unique = true)
+    private String name;
+
+    @NotNull
+    private Date date;
+
+    public Certificate(){}
+
+    public Certificate(String name){
+        this.name = name;
+        this.date = new Date();
+    }
+
+    public String getName() {
```

Comment:
If you don't use these getters and setters you can add them with an annotation

Reply:
Thanks for the tip!